### DISCUSSION TITLE
Heat not graphing below 50%

### DISCUSSION THREAD

**Discussion by russ999** [artisan-roaster-scope/artisan#1020] [General]
Started getting a weird glitch where the heater percentage does not plot on graphs whenever heat is below 50%. Heat shows up as a flat line at 50, unless and until heat goes above 50.  See image attached. Everything is roasting fine and temperature graphs look good. I think it was working a while ago and can't figure it why it stopped working. Any ideas?

![image](https://user-images.githubusercontent.com/91618688/204676432-79d00a0e-a5b4-4149-9693-63f357d1e14b.png)

**Comment by MAKOMO** [comment]
Did you have an input filter set? Check under menu Config >> Curves, 2nd tab, section "Input Filter", "Limits".

**Reply by russ999** [reply]
Thank you!  Duh. I set the filter for temperature and totally forgot about the heater.
